encode.go

```diff
@@ -44,12 +46,19 @@ func getTmpBuffer(w io.Writer, n int) []byte {
 }
 
 func encodeSQLValue(w io.Writer, arg interface{}) error {
+	log.Printf("%T %s", arg, arg)
 	// Use sql.driver to convert the arg to a sql.Value which is simply
 	// an interface{} with a restricted set of types. This also takes
 	// care of using the sql.Valuer interface to convert arbitrary types
 	// into sql.Values.
 	dv, err := driver.DefaultParameterConverter.ConvertValue(arg)
 	if err != nil {
+		// We may be in the presence of a type alias not supported by the
+		// database/driver DefaultParameterConverter. Special handling.
+		value := reflect.ValueOf(arg)
+		if _, ok := baseKinds[value.Kind()]; ok {
+			return encodeSQLValue(w, asKind(value))
+		}
```

Comment:
I know `uint8` works, because we have a system at Square that uses such a type alias. Not sure about others, and wanted to have a complete implementation in squalor rather than rely on various oddities at the lower level.